### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by kesla**
R = @danmakenoise

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+node_modules
+dist
+.nyc_output
```
Comment (stankud) [inline .gitignore]: newline 😄

**Comment by stankud** [comment]
@kesla should we add support for async/await here?

**Comment by kesla** [comment]
@stankud there is support for async/await in here. `fast-async` is async/await but faster than the babel built in version

File: .gitignore (hunk 1 above)
Comment (kesla) [reply .gitignore]: ty

**Comment by kesla** [comment]
aight I'm going to merge and release this and test it on the pwa work we're doing.

**Pull request merged**